pkg/controller/manager/sync.go

```diff
@@ -108,6 +109,12 @@ func (c *Controller) sync(ctx context.Context, key string) error {
 		return nil
 	}
 
+	if !controllerhelpers.IsScyllaClusterRolledOut(sc) {
```

Comment:
does this mean that any time scyllacluster is rolling out or partially degraded, the manager controller stops working?

Reply:
yes

Reply:
given rolling updates leave the cluster HA why should this cripple the manager controller?